frontend/src/pages/mainPage.js

```diff
@@ -23,14 +24,16 @@ function MainPage() {
     const { t } = useTranslations();
     const [news, setNews] = useState([])
     const [newsPaginated, setNewsPaginated] = useState([])
+    const noLatestNews = useBurgerMenu(`${globalVariables.windowSizeForBurger.latestNews}`);
```

Comment:
that's usually not a good idea to name your variables having them deny something (no... without...). It's better to go from positive, e.g. check if something EXISTS, and then, if not the '!<variableName>' makes much more sense and is easier to understand.

So in this case I'd go with "latestNews" or "latestNewsPresent"
something like that.

However, not that I look closer, this variable barely has something to do with news. It is focused on window size, is it not? 

Please rename this variable so that it reflects what it holds